### ISSUE TITLE
ProtocolLib dependancy warning

### ISSUE THREAD

**Issue by B3fuddled** [MassiveCraft/MassiveCore#19]
I'm running ProtocolLib v2.5.0 along with Mcore v6.50, and getting this warning when starting up the server (CB 1.6.2 Dev):

[ProtocolLib] [PacketFilterManager] mcore doesn't depend on ProtocolLib. Check
that its plugin.yml has a 'depend' directive.

The ProtocolLib author stated this:
"This is a new warning I added to detect plugins that are playing it fast and
loose with the packet system. If a plugin is using ProtocolLib, they should
preferably add it to either soft-depend of depend, or you run the risk of a
NullPointerException on startup.

Bottom line, it's a warning. If you haven't noticed any actual errors, you can
safely ignore it. But I'd still suggest contacting the developers of mcore to
get this fixed."

Not sure what to make of this.. is this a problem or not?  Everything seems to run fine though.
